README.rst

```diff
@@ -116,7 +116,7 @@ Features
 - **Preview images**: A preview of image files can be displayed before uploading with browsers supporting the required JS APIs
 - **No browser plugins required**: The implementation is based on open standards like HTML5 and JavaScript and requires no additional browser plugins
 - **Graceful fallback for legacy browsers**: Uploads files via XMLHttpRequests if supported and uses iframes as fallback for legacy browsers
-- **Cross-site file uploads**: Supports uploading files to a different domain with cross-site XMLHttpRequests or iframe redirects
+- **Drag and drop uploads from another web page**: Supports uploading files dragged from one page into another (tested with Firefox and Chrome)
```

Comment:
I know, this is the way it worked before this PR (when it worked.. looks like at master branch didn't work anymore).

I suggest to open a new issue to review for other content types, and try to add the `id` automatically.

This PR is already too big, and the main purpose here is to update `jquery.fileupload` plugin.